apps/web/models.py

```diff
@@ -33,11 +35,13 @@ class Team(models.Model):
 
     name = models.CharField(max_length=100)
     email = models.EmailField(max_length=70)
+    description = RichTextField(null=True)
```

Comment:
You can simply keep it text field instead of using RichtextField. We don't want to use ckeditor just for the purpose of showing Team details.